common/src/main/java/us/myles/ViaVersion/exception/CancelDecoderException.java

```diff
@@ -0,0 +1,34 @@
+package us.myles.ViaVersion.exception;
+
+import io.netty.handler.codec.DecoderException;
+import us.myles.ViaVersion.api.Via;
+
+public class CancelDecoderException extends DecoderException {
```

Comment:
Probably worth javadoc'ing the top of this class to explain that it's used to cancel packets.